### ISSUE TITLE
Alternative CCP - CCPZ

### ISSUE THREAD

**Issue by gd-99** [skx/cpm-fat#23]
Hi SKX, in case you don't get notifications from the root cpm_fat site, thought I would drop an issue here to note the command processor [CCPZ here](https://github.com/z80playground/cpm-fat/issues/70). As I indicate this works well on my board though not fully tested yet.

I would be interested to know if you manage to get CCPZ running on a real Z80 Plaground.

I use the .COM extension rather than .bin - loads through the cpm.cfg file just as well - because in the future I was going to look at creating a CP/M loader to change the CCP on the fly. Shouldn't be to hard? Simply load the new CCP at DE00h and then jump to 0000h?

Anyway hope this works for you?

**Comment by skx** [comment]
Somehow I didn't receive any notifications of this issue, and another closed today.  Weird - I wonder if because it was a forked repository.

Thanks for the explicit pointer, given the lack of activity on the forums I'd not checked on the main playground repository for quite some time - though I did open a couple of trivial pull requests there when I started tinkering.

I'll have a play with your CCP later.  To be honest I wonder if it might be worth packaging as a COM.  As you say I could compile and then append it to a simple .com file as a driver.

      org 100
      ld de, 0xDE00
      ld hl, end_of_file
      ld bc, 4092  ; 4k should be enough for anyway ;)
      ldir
      jp 0x0000
    :end_of_file

That might be a fun way to go, although I guess the transition would be one way!
